### DISCUSSION TITLE
macOS Mail: plugin to create new alias and send email from it

### DISCUSSION THREAD

**Discussion by notDavid** [simple-login/app#1047] [Feature Requests]
Hi there, please consider making a macOS Mail plugin to create new aliases.

Note that this is already possible with Apple's native "Hide my Email" feature.

Thank you!

<img width="501" alt="Screenshot_2022-06-05_at_11_24_27" src="https://user-images.githubusercontent.com/1530401/172044266-2591e5ff-5373-4531-ab1d-8137e09410ff.png">

**Comment by traktuner** [comment]
That would be SO great!

**Comment by JollyRoger8X** [comment]
Not the greatest solution, but a workaround nevertheless:

You can do this manually by opening **Mail > Preferences > Accounts**, selecting the account, clicking **Email Address > Edit Email Addresses**, and adding the SimpleLogin address there. 

**EDIT**: Scratch that. Spoke too soon. I was unable to actually send the message due to an error. 🤨
